tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
   "compilerOptions": {
```

Comment:
nit: can we alphabetize the fields in `compilerOptions`

Reply:
What exactly do you mean by that?

Reply:
the keys in `compilerOptions` could be sorted alphabetically (as it was before)